server/data/localMockData/inPrisonCourse.ts

```diff
@@ -0,0 +1,73 @@
+import { subMonths } from 'date-fns'
+import { InPrisonCourse } from '../../services/interfaces/curiousService/CuriousInPrisonCourses'
+
+const aValidEnglishInPrisonCourse = (): InPrisonCourse => {
```

Comment:
A handful of local mock data files for the various types I've introduced here, all of which represent more realistic data (eg. sensible course titles etc)